salt/modules/logmod.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+'''
+On-demand logging
+=================
+
+.. versionadded:: Nitrogen
+
+The sole purpose of this module is logging messages in the (proxy) minion.
+It comes very handy when debugging complex Jinja templates, for example:
+
+.. code-block:: jinja
+
+    {%- for var in range(10) %}
+      {%- set logged = salt.log.info(var) %}
+    {%- endfor %}
+
+CLI Example:
+
+.. code-block:: bash
+
+    salt '*' log.error 'Please dont do that, this module is not for CLI use!'
```

Comment:
Ah. I understand. Though, I suppose I don't really see the harm in allowing this function to be executed remotely via the CLI. Do you consider it a problem simply because it's outside the intended use or do you see a more serious issue here?